### PR TITLE
Image

### PR DESCRIPTION
**Pull request by talrejah02**
please review, suggestions are welcome.
ps: not responsive yet

### CODE REVIEW

**Review by kshitijthatte — APPROVED** [review]
Some files are repeated here from the previous PR, have given suggestions about them there. Apart from that, it looks good.

File: components/common-css/container.css [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    width: 80% ;
+    padding: 1rem;
+    border-radius: 10px;
+    background-color: var(--card-bg);
+    margin:1rem;
+    box-shadow: var(--box-shadow);
+}
+.doc-container{
+    padding: 1rem;
+    padding-top: 0.5rem;
```
Comment (ShreyashMahajan) [inline components/common-css/container.css:21]: try to use px for less than 1rem

File: components/image/image.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+            <a href="../../index.html" class="nav-link">Home</a>
+            <a href="./components/Avatar/avatar.html" class="nav-link">Docs</a>
+            <a href="https://github.com/talrejah02/component-library" class="nav-link">Github</a>
+        </section>
+    </nav>
+    <section class="component">
+        <aside class="drawer">
+            <section class="drawer-header">
+                <span class="header-title">Get Started</span>
+                <!-- <span class="header-subtitle"></span> -->
```
Comment (ShreyashMahajan) [inline components/image/image.html:30]: try not to keep unwanted comments

**Review by ShreyashMahajan — APPROVED** [review]
The overall code structure is very good, semantic elements are used in the whole code. Just a suggestion that try to keep image size lesser. Liked the UI so much. Good to merge

**Pull request merged**